tutorials/tts/2_TTS_Tacotron2_Training.ipynb

```diff
@@ -1,335 +1,347 @@
-        "...\r\n",
-        "```\r\n",
-        "Please note that the duration is in seconds.\r\n",
-        "\r\n",
-        "Lastly, update the labels inside the Tacotron 2 yaml config if your data contains a different set of characters.\r\n",
-        "\r\n",
-        "Then you are ready to run your training script:\r\n",
-        "```bash\r\n",
-        "python tacotron2.py train_dataset=YOUR_TRAIN.json validation_datasets=YOUR_VAL.json trainer.gpus=-1\r\n",
-        "```"
-      ]
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "2KctbQ61MmHy"
-      },
-      "source": [],
-      "execution_count": null,
-      "outputs": []
-    }
-  ]
-}
\ No newline at end of file
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "wqPMTEXXYUP4"
+   },
+   "source": [
+    "# License\n",
+    "\n",
+    "> Copyright 2020 NVIDIA. All Rights Reserved.\n",
+    "> \n",
+    "> Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "> you may not use this file except in compliance with the License.\n",
+    "> You may obtain a copy of the License at\n",
+    "> \n",
+    ">     http://www.apache.org/licenses/LICENSE-2.0\n",
+    "> \n",
+    "> Unless required by applicable law or agreed to in writing, software\n",
+    "> distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "> WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "> See the License for the specific language governing permissions and\n",
+    "> limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "SUkq9HAvYU7T"
+   },
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies# .\n",
+    "\"\"\"\n",
+    "# # If you're using Colab and not running locally, uncomment and run this cell.\n",
+    "# !apt-get install sox libsndfile1 ffmpeg\n",
+    "# !pip install wget unidecode\n",
+    "# BRANCH = 'r1.0.0rc1'\n",
+    "# !python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[tts]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ZivXzmq0YYLj"
+   },
+   "source": [
+    "# Tacotron2 and NeMo\n",
+    "\n",
+    "Tacotron2 is a neural network that converts text characters into a mel spectrogram. For more details on the model, please refer to Nvidia's Tacotron2 Model Card, or the original [paper](https://arxiv.org/abs/1712.05884).\n",
```

Comment:
Do we have a link for the model card we can add here?

Reply:
I would but nothing is public yet

Reply:
Gotcha. LGTM